internal/mode/static/telemetry/collector_test.go

```diff
@@ -127,21 +139,40 @@ var _ = Describe("Collector", Ordered, func() {
 			Version:             version,
 			PodNSName:           podNSName,
 		})
-		k8sClientReader.GetCalls(createGetCallsFunc(ngfPod, ngfReplicaSet))
+
+		baseGetCalls = createGetCallsFunc(ngfPod, ngfReplicaSet, kubeNamespace)
+		k8sClientReader.GetCalls(baseGetCalls)
 	})
 
+	mergeGetCallsWithBase := func(f getCallsFunc) getCallsFunc {
+		return func(ctx context.Context, nsName types.NamespacedName,
```

Comment:
nit:

If you break the line can you split each parameter on a new line so it looks something like this? (indentation is messed up in the example)

```
return func(
ctx context.Context,
nsName types.NamespacedName,
object client.Object,
option ...client.GetOption,
)
```

Reply:
Its not messed up, just another way of doing it.

Reply:
@salonichf5, please review our style guide: https://github.com/nginxinc/nginx-gateway-fabric/blob/main/docs/developer/go-style-guide.md#consistent-line-breaks

Long lines should be broken up as @bjee19 described.

Reply:
Yes I've updated it.

Reply:
sorry 😓  I meant my example I provided the indentation was messed up since I can't tab in the code block example😅

Reply:
no worries, thank you for pointing it out.